From a8c28f497ad34f05fae70e7f41e0222d0206eb06 Mon Sep 17 00:00:00 2001
From: Kalyani Akula <kalyani.akula@xilinx.com>
Date: Mon, 3 Feb 2020 15:46:01 +0530
Subject: [PATCH 1110/1851] arm64: zynqmp: Added device tree node for
 zynqmp-aes driver

commit 07bc3501e81c322d98805710df355964de1cfe71 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds device node for zynqmp-aes driver

Signed-off-by: Kalyani Akula <kalyani.akula@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
index 0162da6e289f..7415d0801a8b 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
+++ b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
@@ -158,6 +158,10 @@
 					      "aux_ref_clk",
 					      "gt_crx_ref_clk";
 			};
+
+			xlnx_aes: zynqmp_aes {
+				compatible = "xlnx,zynqmp-aes";
+			};
 		};
 	};
 
-- 
2.31.1

